config/locales/ar.yml

```diff
@@ -142,8 +146,12 @@ ar:
     download_files: تحميل الملفات
     download_image: تحميل الصورة
     file_name: اسم الملف
+    hide_tashkeel: إخفاء التشكيل
+    hide_tashkeel_tooltip: إخفاء علامات التشكيل
     pdf_content_only: الكتاب المُصوّر فقط
     search: البحث
+    show_tashkeel: إظهار التشكيل
+    show_tashkeel_tooltip: إظهار علامات التشكيل
```

Comment:
In the video, I can't see that the toggle button tooltip text changes from hide to show and vice-versa. Am I missing something?